corgidrp/l1_to_l2a.py

```diff
@@ -42,27 +49,37 @@ def prescan_biassub(input_dataset, bias_offset=0., return_full_frame=False):
             # Get the image area
             image_data = slice_section(frame_data, obstype, 'image')
             image_dq = slice_section(frame_dq, obstype, 'image')
-            # error maps are 3-D
+            
+            # Special treatment for 3D error array
             image_err = []
-            for err_slice in frame_err:
+            for err_slice in frame_err: 
                 image_err.append(slice_section(err_slice, obstype, 'image'))
             image_err = np.array(image_err)
-            
-            # Get the part of the prescan that lines up with the image, and do a
-            # row-by-row bias subtraction on it
+
+            # Get the part of the prescan that lines up with the image
             i_r0 = detector_areas[obstype]['image']['r0c0'][0]
             p_r0 = detector_areas[obstype]['prescan']['r0c0'][0]
             i_nrow = detector_areas[obstype]['image']['rows']
-     
-            # Get data from prescan (alined with image area)
             al_prescan = prescan[(i_r0-p_r0):(i_r0-p_r0+i_nrow), :]    
-            medbyrow = np.median(al_prescan, axis=1)[:, np.newaxis]
+            
         else:
+            # Use full frame
             image_data = frame_data
-            image_err = frame_err
             image_dq = frame_dq
+
+            # Special treatment for 3D error array
+            image_err = []
+            for err_slice in frame_err: 
+                image_err.append(err_slice)
+            image_err = np.array(image_err)
+
+            al_prescan = prescan
+
+        # Measure bias and error (standard error of the median for each row, add this to 3D image array)
+        medbyrow = np.median(al_prescan, axis=1)[:, np.newaxis]
+        sterrbyrow = np.std(al_prescan, axis=1)[:, np.newaxis] * np.ones_like(image_data) / np.sqrt(al_prescan.shape[1])
```

Comment:
Should this have the sqrt(pi/2) as I mentioned earlier?

Reply:
Ah, I think I misinterpreted your earlier comment, and was accidentally comparing the error - measured using the formula for std error on the mean - to the expected error for the median. It looks like the errors for the standard error on the median are still a bit inflated compared to the expectation. Here are the updated plots using the correct formulas:
![image](https://github.com/roman-corgi/corgidrp/assets/74611610/d6a4130c-ab40-455d-8f4f-1328d73b7636)
![image](https://github.com/roman-corgi/corgidrp/assets/74611610/82fbf17f-4d7d-4073-925b-a4ed5c02e004)
![image](https://github.com/roman-corgi/corgidrp/assets/74611610/3de03de3-fb0c-4344-aca8-d3748cb20874)

Reply:
Hmm, why are the values in the colorbar in the "measured error" plot higher than before. Did something change? I would have thought that if we are still using the median for bias subtraction that the measured error shouldn't change?

Reply:
Yeah that's because I was accidentally using the formula for the mean before. So I fixed that and now the errors are more inflated.

Reply:
But the median bias measurement itself is unchanged